### ISSUE TITLE
incorrect validation cases

### ISSUE THREAD

**Issue by ghost** [zalando/intellij-swagger#40]
Affected Version: 0.0.11
### Properties named 'contact'

In this example, "$ref" would be marked as invalid key:

```
"User": {
    "type": "object",
    "description": "User",
    "properties": {
        "contact": {
            "$ref": "#/definitions/Contact"
        }
    }
}
```

Suggestions for invalid field contain 'email', 'name', and 'url'. Changing property name to something else fixes the issue.
### Array item descriptions

```
"ExampleList": {
    "type": "array",
    "description": "valid",
    "items": {
        "type": "string",
        "description": "invalid key"
    }
}
```

**Comment by ghost** [comment]
Also minor suggestion. Could you include link to this repository in plugin description? It would make it a lot easier to find.

**Comment by semonte** [comment]
Thanks for the report and suggestion. I will fix the reported bugs and add a link to repo in description.

**Comment by semonte** [comment]
I fixed the _contact_ field issue and will make a new release soon. However, I might be missing something for the _items_ field. According the specs _description_ field is not allowed http://swagger.io/specification/#itemsObject

Do you still think there is an issue with this?

**Comment by ghost** [comment]
I must say I haven't noticed that. I was judging by the fact that it works in [swagger editor](http://editor.swagger.io) and is most likely accepted by code generator as well.

Example: [screenshot](http://i.imgur.com/hPblEFN.png)

```
...
definitions:
  ExampleArray:
    type: array
    description: array description
    items:
      type: string
      description: array item description
```

I guess it's up to you in this case. I already moved all array item descriptions to array descriptions in my api definition, so this is no longer an issue for me.

**Comment by semonte** [comment]
I have published version 0.0.12. It contains a fix for the contact field issue. I will leave the description error as it is, since everything else is implemented according to Swagger specification.

Again, thanks for you report and helping to improve the editor!
